lib/plugins/Label_4N.test.ts

```diff
@@ -0,0 +1,133 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_4N } from './Label_4N';
+
+test('matches Label 4N qualifiers', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_4N(decoder);
+
+  expect(decoderPlugin.decode).toBeDefined();
+  expect(decoderPlugin.name).toBe('label-4n');
+});
+
+test('decodes Label 4N variant 1', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_4N(decoder);
+
+  // https://globe.adsbexchange.com/?icao=A15027&showTrace=2024-09-23&timestamp=1727057017
+  const text = '22024N  MCI  JFK1\r\n0013  0072 N040586 W074421   230';
+  const decodeResult = decoderPlugin.decode({ text: text });
+
+  expect(decodeResult.decoded).toBe(true);
+  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.name).toBe('label-4n');
+  expect(decodeResult.formatted.description).toBe('Airline Defined');
+  expect(decodeResult.message.text).toBe(text);
+  expect(decodeResult.raw.day_of_month).toBe('22');
+  expect(decodeResult.remaining.text).toBe('02 0013  0072');
+  expect(decodeResult.formatted.items.length).toBe(4);
+  expect(decodeResult.formatted.items[0].code).toBe('ORG');
+  expect(decodeResult.formatted.items[0].value).toBe('MCI');
+  expect(decodeResult.formatted.items[1].code).toBe('DST');
+  expect(decodeResult.formatted.items[1].value).toBe('JFK');
+  expect(decodeResult.formatted.items[2].code).toBe('POS');
+  expect(decodeResult.formatted.items[2].value).toBe('40.977 N, 74.702 W');
+  expect(decodeResult.formatted.items[3].code).toBe('ALT');
+  expect(decodeResult.formatted.items[3].value).toBe('23000 feet');
+});
+
+test('decodes Label 4N variant 2B', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_4N(decoder);
+
+  // https://app.airframes.io/messages/3421601874
+  const text = '285,B,69005074-507,10/12,+36.081,-094.810,35014,002.3,ELP,SDF,SDF,17R/,17L/,0,0,,,,,,0,0,0,0,1,,,,,247.0,014.2,261.2,421A';
```

Comment:
do we know what the 285 is in these? I'm not seeing it in the tests

Reply:
not sure but I haven't seen it be anything else so I assumed it was a constant value and skipped it